mitodl/micromasters
# Added course contact email link to the student dashboard

#### What are the relevant tickets?

Closes #2284 
⚠️️ REBASED ON #2452 ⚠️️

#### What's this PR do?

Adds the course contact email link to the student dashboard

#### Where should the reviewer start?

Probably static/js/containers/DashboardPage.js

#### How should this be manually tested?

1. Seed the DB and set your staff user
1. Set `MAILGUN_RECIPIENT_OVERRIDE` to your email
1. Set any/all 'Analog Learning' courses to have a `contact_email`
1. Enroll in the courses that you've set a contact email for (`alter_data` can be used)
1. Reindex
1. Check the student dashboard and send an email using the "Contact Course Team" link
1. Check that the email you've received is correct

#### Any background context you want to provide?

One of the features that #2284 calls for is a message to unpaid users that they can only contact the course team once they've paid (https://github.com/mitodl/micromasters/issues/2284#issuecomment-274866731). I don't have the graphic for that modal, and this PR is sizable enough already that I think it makes sense to create a new issue to implement that new message/modal. 

#### Screenshots (if appropriate)

<img width="1061" alt="ss 2017-01-31 at 07 36 50" src="https://cloud.githubusercontent.com/assets/14932219/22465102/1474a1b4-e788-11e6-9efd-9dd5585524d6.png">
